lib/fhir_client/client.rb

```diff
@@ -170,10 +170,10 @@ def capability_statement(format = @default_format)
     # Method returns a conformance statement for the system queried.
     # @return
     def conformance_statement(format = @default_format)
-      if @cached_conformance.nil? || format != @default_format
+      if @cached_capability_statement.nil? || format != @default_format
```

Comment:
Huh... I sort of like the `@` -- makes it clear what we're dealing with here. _Shrugs_